themes/default/content/resources/getting-started-with-aws/index.md

```diff
@@ -77,8 +77,8 @@ main:
     presenters:
         - name: "Kat Cosgrove"
           role: "Staff Developer Advocate, Pulumi"
-        - name: "Sophia Parafina"
-          role: "Technical Marketing Manager"
+        - name: "Matty Stratton"
+          role: "Staff Developer Advocate, Pulumi"
```

Comment:
I think I'm the one presenting this time for AWS for Cloud Engineering Summit, and I think Marina from AWS is joining me. Is this one form for all of the upcoming AWS workshops? Should I just add us both here?

Reply:
👍

Reply:
@zchase I think the name on this should be Laura Santamaria